pkg/reconciler/pipelinerun/controller.go

```diff
@@ -31,7 +32,7 @@ import (
 	pipelinerunreconciler "github.com/tektoncd/pipeline/pkg/client/injection/reconciler/pipeline/v1beta1/pipelinerun"
 	resourceinformer "github.com/tektoncd/pipeline/pkg/client/resource/injection/informers/resource/v1alpha1/pipelineresource"
 	"github.com/tektoncd/pipeline/pkg/reconciler"
-	"github.com/tektoncd/pipeline/pkg/reconciler/pipelinerun/config"
```

Comment:
Is this package still used ? (if not we should remove it :angel: )

Reply:
Oh, I was sure I removed that... Will do now.
It looks like the first import wins, it wouldn't work when using the second. I'm surprised neither the compiler nor the linter catches that?

Reply:
oh, never mind, I was looking on the phone, I thought this on the right side of the diff :P

Reply:
I removed the package completely now.
It was not really used for a while, I couldn't find a place in the code where it was used going back in the code for a while. `InitializeArtifactStorage` always took a k8s client as input and no context, so there's no way it could access the store at any point.